### ISSUE TITLE
GRIPSS Filtering Error for MHC Contigs

### ISSUE THREAD

**Issue by DarioS** [hartwigmedical/hmftools#106]
GRIPSS is failing for HLA genes' contigs, which have weird names such as:

```
##contig=<ID=HLA-A*23:01:01,length=3502>
##contig=<ID=HLA-A*23:09,length=3104>
##contig=<ID=HLA-A*23:38N,length=3020>
##contig=<ID=HLA-A*24:02:01:01,length=3502>
##contig=<ID=HLA-A*24:02:01:02L,length=3502>
##contig=<ID=HLA-A*24:02:01:03,length=3075>
##contig=<ID=HLA-A*24:02:03Q,length=3247>
##contig=<ID=HLA-A*24:02:10,length=3356>
##contig=<ID=HLA-A*24:03:01,length=3502>
##contig=<ID=HLA-A*24:07:01,length=3502>
##contig=<ID=HLA-A*24:08,length=3502>
##contig=<ID=HLA-A*24:09N,length=3502>
##contig=<ID=HLA-A*24:10:01,length=3502>
##contig=<ID=HLA-A*24:11N,length=3503>
##contig=<ID=HLA-A*24:152,length=3176>
##contig=<ID=HLA-A*24:20,length=3502>
##contig=<ID=HLA-A*24:215,length=3116>
##contig=<ID=HLA-A*24:61,length=3043>
##contig=<ID=HLA-A*24:86N,length=3415>
##contig=<ID=HLA-A*25:01:01,length=2917>
```

From the Java traceback, it looks like it is removing all text after colons.

```
Exception in thread "main" java.lang.IllegalStateException: contig HLA-A*24 not defined
        at com.hartwig.hmftools.gripss.ContigComparator.compare(ContigComparator.kt:47)
        at com.hartwig.hmftools.gripss.ContigComparator.compare(ContigComparator.kt:33)
```

**Comment by jonbaber** [comment]
Thanks for the feedback Dario. Monday is a public holiday here but I will
fix this up Tuesday.

If you want to keep testing in the meantime I recommend using BCFTools to
filter the standard contigs.

**Comment by jonbaber** [comment]
Fixed in 1.3
